src/types/Navigation.ts

```diff
@@ -52,6 +52,9 @@ export enum ScreenName {
   OParlSearch = 'OParlSearch',
   Service = 'Service',
   Settings = 'Settings',
+  SUEList = 'SUEList',
+  SUEMap = 'SUEMap',
+  SUEMapScreen = 'SUEMapScreen',
```

Comment:
i suggest using always Sue instead of SUE for easier readability

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/960/commits/754a44edc67676d05ca82e810e413c9d9a8c8e77